workflows/utilities/wf_merlin_magic.wdl

```diff
@@ -366,6 +367,10 @@ workflow merlin_magic {
           srst2_min_edge_depth = srst2_min_edge_depth,
           srst2_gene_max_mismatch = srst2_gene_max_mismatch
       }
+    # this is nested inside of conditional check for merlin_tag so should only be available for Vibrio samples (meaning srst2 task was run)
```

Comment:
and now I'm considering moving it inside of the conditional `if (!assembly_only && !ont_data)`...

Reply:
I cannot read, the check is already nested inside of !assembly_only and !ont_data conditional 

Just going to update the comment to clarify